.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "2.6"
```

Comment:
we should really add python3

Reply:
Sure, but I'm not going to do that in this PR

Reply:
cool